<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Generation_QrCode</class>
 <widget class="QWidget" name="Generation_QrCode">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>943</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generation_QrCode</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QGridLayout" name="global">
     <item row="7" column="1">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="2" colspan="5">
      <widget class="QLabel" name="label_warning">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color:#FFBD00;</string>
       </property>
       <property name="text">
        <string>Code à usage strictement confidentiel. 
Merci de le garder dans un endroit sécurisé à l'abris des regard indiscret !</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="5" colspan="2">
      <widget class="QPushButton" name="bouton_imprimer">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Imprimer</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/imprimante.png</normaloff>Images/imprimante.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButton">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string> Mode d'emploi</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/mode_d'emploi.jpeg</normaloff>Images/mode_d'emploi.jpeg</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="QPushButton" name="pushButton_2">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
         <weight>75</weight>
         <italic>true</italic>
         <bold>true</bold>
         <kerning>false</kerning>
        </font>
       </property>
       <property name="text">
        <string>Attention</string>
       </property>
       <property name="icon">
        <iconset>
         <normaloff>Images/warning.ico</normaloff>Images/warning.ico</iconset>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="2" rowspan="2" colspan="5">
      <widget class="QLabel" name="label_mode_demploi">
       <property name="font">
        <font>
         <family>Comic Sans MS</family>
        </font>
       </property>
       <property name="text">
        <string>  Appuyez sur la touche impréssion. Sélectionnez l'imprimante. Cliquez sur print. Une feuille avec le 
 QrCode unique du salarié sera générée.
 
 Le salarié n'aura plus qu'à scanner ce QrCode pour avoir sa clef d'indentification unique lui servant à ouvrir 
 les portes auquelles il a accès</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLabel" name="label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2" rowspan="6" colspan="4">
      <widget class="QWidget" name="zone_QrCode" native="true">
       <layout class="QGridLayout" name="QrCode_2">
        <item row="0" column="1">
         <widget class="QLabel" name="label_qrcode">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
